resources/phing/AtoumTask.php

```diff
@@ -287,13 +309,37 @@ public function getCodeCoverage()
 		return $this->codeCoverage;
 	}
 
+	public function setTelemetry($telemetry)
+	{
+		$this->telemetry = (boolean) $telemetry;
+
+		return $this;
+	}
+
+	public function getTelemetry()
```

Comment:
`TelemetryIsEnabled()` to be consistent with what we do in other class (e.g https://github.com/atoum/atoum/blob/master/classes/test.php#L719)